code-styling/markdown/README.md

```diff
@@ -0,0 +1,1155 @@
+  ```
+
+  _Incorrect examples_:
+  
+  ```Markdown
+  [Barcode Overview](http://www.tec-it.com/download/PDF/Barcode_Reference_EN.pdf)
+  [Code Connected Volume 1](http://hintjens.wdfiles.com/local--files/main:files/cc1pe.pdf) - Pieter Hintjens(PDF)
+  ```
+
+- If the book or document is available in multiple formats, it is RECOMMENDED to add linked additional formats in the
+  format specification.
+  
+  _Correct example_:
+  
+  ```Markdown
+  [A Unix Person's Guide to PowerShell][powershellpdf] - Matt Penny (PDF, [ePub][powershellepub])
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  [A Unix Person's Guide to PowerShell][powershellpdf] - Matt Penny (PDF)
+  [A Unix Person's Guide to PowerShell][powershellepub] - Matt Penny (ePub)
+  ```
+  
+## 9. Images
+
+- Images MUST follow the same syntax as links, but preceded with an exclamation (`!`) character.
+- Images MUST follow all same rules as links.
+- Links to internal images MUST use the inline link style and use relative paths.
+- Links to external images MUST use the reference link style.
+
+_Examples_:
+
+```Markdown
+![alt text](image.png "Image title")
+
+[![Google][first google logo]][google]
+
+[google]: http://www.google.com "Google Search"
+[first google logo]: https://en.wikipedia.org/wiki/Google_logo#/media/File:First-google-logo.gif "Google Logo"
+```
+
+## 10. Tables
+
+- Tables MUST NOT be indent, unless the table is placed inside a list.
+- Tables MUST be preceded and followed by a newline.
+- Tables MUST be fenced with pipes (`|`).
+- The table opening pipe (`|`) MUST be followed by a space character.
+- The table closing pipe (`|`) MUST be preceded by a space character.
+- Table header and body MUST be separated using at least trhee hyphens (`---`), wider column MUST match column width.
+- Pipe characters, delimiting columns, must be preceded and followed by a space characte
+- All pipe characters in a table MUST be align vertically.
+- Column width SHALL be determined by the longest cell in the column.
+
+_Correct example_:
+
+```Markdown
+| Tables  | Are     | Cool  |
+| ------- | ------- | ----- |
+| apples  | a lot   | $1600 |
+| bananas | a dozen | $12   |
+| grapes  | one     | $1    |
+```
+
+_Incorrect example_:
+
+```Markdown
+Tables | Are | Cool
+--- | --- | ---
+apples | a lot | $1600
+bananas | a dozen | $12
+grapes | one | $1
+```
+
+- Tables column contents MAY be aligned using the semicolon (`:`) in the body and header separator.
+
+  _Example_:
+  
+  ```Markdown
+  | Left    | Centered | Right |
+  | :------ | :------: | ----: |
+  | apples  | a lot    | $1600 |
+  | bananas | a dozen  | $12   |
+  | grapes  | one      | $1    |
+  ```
+
+## 11. Emoji
+
+- It is RECOMMENDED to use emoji, they help convey tone and intent, both of which are frequently lost in translation
+  when communicating online.
+
+- In order to maintain maximum compatibility with different platform, used emoji MUST be listed on the emoji cheat 
```

Comment:
Rule that applies here:
- Links to websites SHOULD NOT use automatic links, unless the full link is displayed.

I wanted to display the full url to this website. Since one could simple remember this link for easy future reference. That's why I placed it as an automatic link and displayed it a little prominent on its own line.